virttest/qemu_devices/qcontainer.py

```diff
@@ -302,6 +303,9 @@ def _probe_capabilities(self):
         # -object tdx-guest
         if self.has_object('tdx-guest'):
             self.caps.set_flag(Flags.TDX_GUEST)
+        # -global isa-fdc.drive?=$drive
```

Comment:
Indeed, I originally wrote the old configuration since this line was supposed to preserve compatibility with older versions. I assume then the right phrasing of the comments (there is no written rule for it) is to mention the new handling.